build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     testCompile('org.springframework.boot:spring-boot-starter-test')
     compileOnly('org.projectlombok:lombok')
     annotationProcessor('org.projectlombok:lombok')
+    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.9'
```

Comment:
co to jest i do czego służy?

Reply:
dzięki tej zależności w klasie User w konstruktorze metoda przypisuje losowy userKey